docs/source/config.json

```diff
@@ -10,7 +10,23 @@
     "Migrating to v1.0": "/migrations/1.0",
     "Project Configuration": "/project-configuration",
     "Tutorial": {
-      "Code Generation": "/tutorial/codegen-getting-started"
+      "Code Generation": "/tutorial/codegen-getting-started",
```

Comment:
This sidebar looks kind of awkward to me now in the deploy preview. We have a full "build a sample application tutorial as well as a step-by-step guide for setting up code generation in your own project. 

We also have an entire section on Code Generation itself. I liked having the code generation tutorial separated from the conceptual documentation on code gen, but I'm not sure how to best organize this now. @StephenBarlow any suggestions?